circles/templates/circles/circle_form.html

```diff
@@ -24,11 +24,11 @@
                                placeholder="Circle name"
                                value="{{ circle.name }}"
                                required>
-                        <button class="btn btn-raised btn-material-light-blue" type="submit">Save</button>
-                        <a class="btn btn-raised btn-danger"
-                           href="{% url 'circles:delete' pk=circle.pk %}">Delete</a>
+                        <button class="btn btn-raised btn-primary pull-right" type="submit">Save</button>
```

Comment:
I put the button to the left because that's how the buttons in profile-edit are aligned.

It would look odd if the user switches between profile-edit and circle-manage in the settings card, and the buttons would be aligned differently.

However, I have no preference if all buttons are at the left or right. If you want to keep this one at the right, please also change the buttons in profile-edit.